src/routes/movies.py

```
@@ -0,0 +1,431 @@
+            "content": {
+                "application/json": {
+                    "example": {"detail": "Movie with the given ID was not found."}
+                }
+            },
+        },
+    },
+    status_code=204
+)
+async def delete_movie(
+        movie_id: int,
+        db: AsyncSession = Depends(get_db),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)